modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/session.js

```diff
@@ -538,58 +559,76 @@ AUI.add(
 					return time;
 				},
 
-				_onHostSessionStateChange(event) {
+				_getBanner() {
 					var instance = this;
 
-					if (event.newVal === 'warned') {
-						instance._beforeHostWarned(event);
-					}
-				},
+					var banner = instance._banner;
 
-				_renderBanner(config) {
-					var instance = this;
-
-					var remainingTimeFormatted = instance._formatTime(
-						config.remainingTime
-					);
-
-					instance._updateRemainingTimeTitle(remainingTimeFormatted);
-
-					Liferay.Util.openToast(
-						Object.assign(config, {
-							message:
-								config.remainingTime >= 0
-									? Lang.sub(instance._warningText, [
-											remainingTimeFormatted,
-									  ])
-									: instance._expiredText,
-							messageType: 'html',
-							onClick(eventPayload) {
-								var event = eventPayload.event;
-								var onClose = eventPayload.onClose;
+					if (!banner) {
+						var openToast = instance.get('openToast');
+						var toastId = 'sessionToast';
 
+						var toastDefaultConfig = {
+							onClick: ({event}) => {
```

Comment:
Pro-Tip™: totally optional, but you could use object concide methods here (value of `this` not needed, so no need for lexical scoping of `this` provided by the arrow). ie.

```js
var toastDefaultConfig = {
    onClick({event}) {
        // stuff...
    },    
    onClose({event}) {
        // blah...
    },
    // etc
};
```

Reply:
ugh, yeah, you're right